stl/inc/iterator

```diff
@@ -841,7 +841,7 @@ private:
     public:
         explicit _Proxy(iter_reference_t<_Iter>&& _Right) noexcept(
             is_nothrow_move_constructible_v<iter_value_t<_Iter>>) // strengthened
-            : _Keep(_STD move(_Right)) {}
+            : _Keep(_STD forward<iter_reference_t<_Iter>>(_Right)) {}
 

```

Comment:
If `_Proxy` is to be shared between postfix `operator++` and `operator->`, it needs an `operator*`.